src/main/java/edu/oakland/arttour/dao/ArtTourDAO.java

```diff
@@ -15,20 +16,262 @@
 
   @Autowired private JdbcTemplate jdbcTemplate;
 
-//   public List<Artwork> getGeneralCollection() throws DataAccessException {
-//     return jdbcTemplate.query(Constants.GET_COLLECTION, new Object[], Artwork.mapper);
-//   }
+  public void addAdmin(String email) {
+    jdbcTemplate.update(Constants.ADD_ADMIN, email);
+  }
+
+  public void addArtwork(
+      String artworkId,
```

Comment:
Is there a way to pass the `Artwork` class as  param instead of each field individually? Or is it not possible since we have `location` and `creator`? Is that possible at all in the jdbc template? 
I guess if we had an `Artwork` without `location` and `creator` and have a `Collection` class which extends artwork which has `location` and `creator` could work. 
I'm fine keeping it this way just looking to learn other ways we could also do things.

Reply:
yeah, we could do it the way you suggested. to have artwork by itself, and have a collection class that combines the three.

it would take some extra work. theoretically it could get done. but i think we should leave it the way that it is if you're ok with it, bc time constraints

Reply:
Thank you for explaining that and I'm okay with it. Just looking for 'nice to haves'